modules/ROOT/pages/os-extensions.adoc

```diff
@@ -9,10 +9,11 @@ To do this, you can use https://coreos.github.io/rpm-ostree/[`rpm-ostree install
 To start the layering of a package, you need to write a systemd unit that executes the `rpm-ostree` command to install the wanted package(s).
 By default with `rpm-ostree install`, changes are queued for the next boot.  The `-A/--apply-live` option can be used to apply changes live *and* have them persist.
 
-== Example: Layering nano and setting it as the default editor
+== Example: Adding vim and setting it as the default editor
```

Comment:
Not sure if I prefer `Layering vim` to `Adding vim` here, but just wanted to bring it up as something that crossed my mind.

Reply:
Reverted to layering to keep things explicit